ivadomed/utils.py

```diff
@@ -313,27 +325,30 @@ def run_eval(self):
         return dct
 
 
-def save_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False, unet_3D=False, binarize=True):
-    """Save the prediction as nii.
-        1. Reconstruct a 3D volume out of the slice-wise predictions.
-        2. Re-orient the 3D array accordingly to the ground-truth orientation.
+def pred_to_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False, kernel_dim='2d', bin_thr=0.5):
+    """Convert the NN predictions as nii.
 
-    Inputs:
-        data_lst: list of the slice-wise predictions.
-        z_lst: list of the slice indexes where the inference has been performed.
-              The remaining slices will be filled with zeros.
-        fname_ref: ground-truth fname
-        fname_out: output fname
-        slice_axis: orientation used to extract slices (i.e. axial, sagittal, coronal)
-        debug:
+    Based on the header of fname_ref image, it creates a nibabel object from the NN predictions (data_lst),
+    given the slice indexes (z_lst) along slice_axis.
 
-    Return:
+    Args:
+        data_lst (list of np arrays): predictions, either 2D slices either 3D patches.
+        z_lst (list of ints): slice indexes to reconstruct a 3D volume.
+        fname_ref (string): Filename of the input image: its header is copied to the output nibabel object.
+        fname_out (string): If not None, then the generated nibabel object is saved with this filename.
+        slice_axis (int): Indicates the axis used for the 2D slice extraction: Sagittal: 0, Coronal: 1, Axial: 2.
+        debug (bool): Display additional info used for debugging.
+        kernel_dim (string): Indicates the number of dimensions of the extracted patches. Choices: '2d', '3d'.
```

Comment:
i would use int, but no big deal

Reply:
scrap that-- i just saw [this discussion](https://github.com/neuropoly/ivado-medical-imaging/pull/155#discussion_r390760531), and on top of that SCT is already using CLI's following convention: `-kernel {2d,3d}`. So using these strings should be fine.